src/FullBuild/Generators/Packagers.fs

```diff
@@ -0,0 +1,24 @@
+﻿module Generators.Packagers
+
+open System.IO
+
+type Nuget = FSharp.Data.XmlProvider< "Examples/Library.nuspec", Global=true, SampleIsList=true >
+
+let private getDependencyVersion package =
```

Comment:
There is already a function either in Parsers/Packages.fs or Parsers/MsBuild.fs that does this.

Could that be possible to merge both of them ?